.github/CODEOWNERS

```diff
@@ -96,13 +96,13 @@
 /sdk/remoterendering/                               @MalcolmTyrrell
 
 # PRLabel: %Schema Registry
-/sdk/schemaregistry/                                @arerlend @sjkwak
+/sdk/schemaregistry/                                @sjkwak @srnagar
```

Comment:
Add @conniey as the primary owner of Schema Registry.

Reply:
I'll do this in a follow-up PR